src/api.js

```diff
@@ -15,7 +15,17 @@ export const getTopics = () => {
 };
 
 export const getArticlesByTopic = (topic) => {
-  return axios.get(
-    `https://nc-news20.herokuapp.com/api/articles/?topic=${topic}`
-  );
+  return axios
+    .get(`https://nc-news20.herokuapp.com/api/articles/?topic=${topic}`)
```

Comment:
You can put in a second argument in this .get:

.get(`https://nc-news20.herokuapp.com/api/articles', {
params: {
topic
}
}

 - this is a better way to do it and means you only need 1 getArticles function